weatherman/weatherman.js

```diff
@@ -0,0 +1,116 @@
+import { getMinMaxObj, getAvgObj } from "./utils/mathUtils.js"
+import { getYear, formattedDateForOutput, getShortMonthFromString } from "./utils/dateUtils.js"
+import { FilReader } from "./fileReader.js"
+
+/**
+ * WeatherMan class .
+ */
+export class WeatherMan {
+
+    /**
+     * Function Constructor.
+     * @param {String} directoryPath Path where files are stored.
```

Comment:
we should add a hyphen after the variable name so the description does not get mixed with the variable name